app/controllers/account/projects_controller.rb

```diff
@@ -8,6 +8,9 @@ def show
     @workspace = parent
     @project = @workspace.projects.find(params[:id])
     @tasks = @project.tasks.row_order_asc
+    @commentable = @project
```

Comment:
no need to create a duplicate variables
just use one, `@project` sounds pretty ok as we are within a projects controller